test/file_picker_macos_test.dart

```diff
@@ -1,4 +1,5 @@
 @TestOn('mac-os')
```

Comment:
The `TestOn()` annotation was reporting that it needed to be on a `library;` directive. This is due to a new change in the language, relating to unnamed libraries I presume. I fixed all the violations in this PR.